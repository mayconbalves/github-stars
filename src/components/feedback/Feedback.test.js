import { render, screen } from '@testing-library/react'
import Feedback from './'

describe('<Feedback />', () => {
  let props = {
    message: 'Not Found'
  }

  it('should render message', () => {
    const { container } = render(<Feedback {...props} />)
    expect(screen.getByText(/Not Found/i)).toBeInTheDocument()
    expect(container.firstChild).toMatchSnapshot()
  })
})
