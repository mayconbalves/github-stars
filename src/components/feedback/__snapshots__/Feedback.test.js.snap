// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Feedback /> should render message 1`] = `
<div
  class="sc-bczRLJ hBXA-DS"
>
  <div
    class="sc-gsnTZi iFAjSo"
  >
    <img
      alt="Not Found"
      class="sc-hKMtZM jFNCiu"
      src="not_found.png"
    />
    <h1
      class="sc-dkzDqf kFxjqL"
    >
      Not Found
    </h1>
  </div>
</div>
`;
